### ISSUE TITLE
caldavzap, roundcube and baikal, possible?

### ISSUE THREAD

**Issue by amclellan** [Offerel/roundcube_caldavzap#13]
I have installed caldavzap as plugin to roundcube and the interface has appeared, but blank. See below:

![image](https://user-images.githubusercontent.com/29105903/74454965-9327ed00-4e7c-11ea-862c-aa082c92f673.png)

When I check the browser console it suggests I have a "401 Unauthorized" error. Detail below:

Error: [loadConfiguration: 'GET https://mail.example.com/plugins/caldavzap/caldavzap_0.13.2/auth/'] code: '401' status: 'error'

Clearly I have missed something important.  Please can anyone help with what?

Many thanks,

theDuck

**Comment by Offerel** [comment]
It seems that you cant authenticate to your DAV backend. There could be several reasons for this behavior. First i would recommend that you check the error messages on Webserver Level and then DAV, if any 401 errors are raised. Additionally please check that the URL you entered in the config points to the right path of your DAV backend and that the DAV username/password is the same as your Roundcube credentials. Also please make sure that the DAV backend is not configured as a Cross-Domain-Setup.

**Comment by amclellan** [comment]
After another long night, I discovered the answer. My Baikal server was configured to accept "Digest" authentication. When this was changed to "Basic", the caldavzap interface came to life.

I have to say, it looks excellent and seems to work well. I was particularly interested to see that it handled my first shared calendar, which had been created manually without the support of Baikal (at this stage). But that is a different story.

Many thanks

theDuck
